src/clang_parser.cpp

```diff
@@ -225,7 +251,7 @@ void ClangParser::parse(ast::Program *program, StructMap &structs)
 
         if (clang_getCursorKind(c) == CXCursor_FieldDecl)
         {
-          auto struct_name = get_clang_string(clang_getCursorSpelling(parent));
+          auto struct_name = get_parent_struct_name(c);
```

Comment:
to minimize the diff, I decided to replace the call here rather than duplicate the logic below.

For non-anonymous structs it should give the same results it always has.